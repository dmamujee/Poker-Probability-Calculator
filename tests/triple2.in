4 S
11 D
6 C
11 H
8 S
11 C
2 S