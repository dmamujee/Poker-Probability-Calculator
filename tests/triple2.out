Hand Ranking: Three of a Kind
